client/components/events/index.ts

```diff
@@ -1 +1 @@
-export { EventList } from "./EventList";
+export { EventViewToggle } from "./EventViewToggle/EventViewToggle";
```

Comment:
I'm guessing it was a conscious choice to call both the folder and the file `EventViewToggle`. I haven't run into that before. What other files would go in that folder?

Reply:
None, I assume. Just didn't want to leave a file without a nice folder (also a habit picked up from my folder-centric last co-op). But honestly, don't mind removing the folder.

Reply:
It seems like bloat to me (which becomes apparent in these long import paths), but that's a matter of taste. @jeffzh4ng thoughts?